presto-main/src/main/resources/webapp/assets/query.js

```diff
@@ -124,9 +123,28 @@ var TaskList = React.createClass({
         }.bind(this));
 
         return (
-            <Table id="tasks" className="table table-striped sortable" sortable={ true } defaultSort={ {column: 'id', direction: 'asc'} }>
+            <Table id="tasks" className="table table-striped sortable" sortable=
+                {[
+                    {
+                        column: 'id',
+                        sortFunction: this.compareTaskId
+                    },
+                    'host',
+                    'state',
+                    'splitsPending',
+                    'splitsRunning',
+                    'splitsDone',
+                    'rows',
+                    'rowsSec',
+                    'bytes',
+                    'bytesSec',
+                    'elapsedTime',
+                    'cpuTime',
+                    'bufferedBytes',
+                ]}
+                defaultSort={ {column: 'id', direction: 'asc'} }>
                 <Thead>
-                        <Th column="id" sortFunction={this.compareTaskId}>ID</Th>
```

Comment:
The way it was done is not mentioned in the official documentation for Reactable and didn't work in my tests. After my change it works.
I'll remove the newline.